modules/templates/proc-creating-templates.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * admin/assembly-admin-templates.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-creating-templates_{context}"]
+= Creating a template by using the Template Editor
+
+You can create a template by using the Template Editor. You can access the Template Editor by opening the URL `\https://<rhdh_url>/create/edit`, or by accessing it in the {product} console.
+
+image::rhdh/template-editor.png[]
+
+.Procedure
+
+. Access the Template Editor by using one of the following options:
+** Open the URL `\https://<rhdh_url>/create/edit` for your {product} instance.
+** Click *Create...* in the navigation menu of the {product} console, then click the overflow menu button and select *Template editor*.
+. Click *Edit Template Form*.
+. Optional: Modify the YAML definition for the parameters of your template. For more information about these parameters, see <<Creating a template as a YAML file>>.
+. In the *Name ** field, enter a unique name for your template.
+. From the *Owner* drop-down menu, choose an owner for the template.
+. Click *Next*.
+. In the *Repository Location* view, enter information about the the hosted repository to publish the template to:
+.. Select an available *Host* from the drop-down menu.
++
+--
+[NOTE]
+====
+Available hosts are defined in the YAML parameters by the `allowedHosts` field:
+
+.Example YAML
+[source,yaml]
+----
+# ...
+        ui:options:
+          allowedHosts:
+            - github.com
+# ...
+----
+
+====
+--
+
+.. In the *Owner ** field, enter an organization, user or project that the hosted repository belongs to.
```

Comment:
```suggestion
.. In the *Owner* field, enter an organization, user or project that the hosted repository belongs to.
```

Reply:
Same as above, the field is actually titled _Owner *_ in the console